utils/meta.mk

```diff
@@ -3,15 +3,10 @@
 VERSION ?= $(shell git rev-parse --short HEAD)
 CURRENT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 
-IMAGE_NAME_WEBAPP=eden-spending-webapp
```

Comment:
where are the image names being sourced?

Reply:
I send in the job to build the image and the job for buil the kubernetes file, this for has a image for dev and another for prod.